readme.md

```diff
@@ -83,29 +74,28 @@ pify(request, {multiArgs: true})('http://sindresorhus.com').then(result => {
 
 Type: `array`
 
-*Works for `pify.all()` only.*
-
 Pick which methods in a module to promisify. Remaining methods will be left untouched.
 
+You can specify either `strings` or `regular expressions` as method names.
+
 ##### exclude
 
-Type: `array`
+Type: `array`  
+Default: `[/^.+Sync$/]`
 
-*Works for `pify.all()` only.*
+Pick which methods in a module **not** to promisify. Methods with names ending with `'Sync'` are excluded by default.
 
-Pick which methods in a module **not** to promisify.
+You can specify either `strings` or `regular expressions` as method names.
 
 ##### excludeMain
 
 Type: `boolean`  
 Default: `false`
 
-*Works for `pify.all()` only.*
-
 By default, if given `module` is a function itself, this function will be promisified. Turn this option on if you want to promisify only methods of the module.
 
 ```js
-const pify = require('pify');
+import pify from 'pify';
```

Comment:
Hmm, was It always like this or you've changed your mind? Because I clearly remember thinking: "I saw Sindre using `import` in readmes, so I should do so too". Now I'm looking throug your repos and it just proves me wrong :confounded: